plugin/src/test/java/org/opensearch/ml/task/TaskRunnerTests.java

```diff
@@ -46,6 +49,8 @@ public class TaskRunnerTests extends OpenSearchTestCase {
     MLTaskDispatcher mlTaskDispatcher;
     @Mock
     MLCircuitBreakerService mlCircuitBreakerService;
+    @Mock
```

Comment:
Curiosity question: this `CuslterService mock has to be used together with extending `OpenSearchTestCase` right?

Reply:
Answering my own question here: no it doesn't have to